src/transformations/correction/closed_ring.jl

```diff
@@ -48,26 +48,21 @@ struct ClosedRing <: GeometryCorrection end
 
 application_level(::ClosedRing) = GI.PolygonTrait
 
-function (::ClosedRing)(::GI.PolygonTrait, polygon)
-    exterior = _close_linear_ring(GI.getexterior(polygon))
-
+function (::ClosedRing)(::Type{T}, ::GI.PolygonTrait, polygon) where T
```

Comment:
especially here I'm not sure if this is necessary, since we're directly copying the point.  Maybe enforcing standard point datatypes could be another fix, though!

Reply:
The function was type unstable before since once branch of the LienarRing dispatch used `tuples` and the other didn't. So I just changed it so all branches use `SVPoints`.

Reply:
Ah I meant the addition of `T` here.  Changing to svpoints does make sense.

Reply:
Then I would need to do eltype or get rid of the `Float64` assumption or else it will just default anyways. Do we want to do that?